app/components/display_devices_ordered_component.rb

```diff
@@ -1,13 +1,13 @@
-class DisplayAllocationsComponent < ViewComponent::Base
+class DisplayDevicesOrderedComponent < ViewComponent::Base
   attr_reader :school
 
   def initialize(school:)
     @school = school
   end
 
-  def allocations
-    std_devices_count = school.std_device_allocation&.raw_allocation.to_i
-    coms_devices_count = school.coms_device_allocation&.raw_allocation.to_i
+  def devices_ordered
+    std_devices_count = school.std_device_allocation&.devices_ordered || 0
+    coms_devices_count = school.coms_device_allocation&.devices_ordered || 0
```

Comment:
Should this (and the one above) use `raw_devices_ordered` instead?

Reply:
(`devices_ordered` will show the amount from the `VirtualCapPool`, if applicable – maybe that's intended?)

Reply:
The column the devices ordered is displayed in is only shown when the RB does not have a virtual cap pool, so I think we're alright. I'm not clear on `raw_devices_ordered` so please let me know if you think this is still a problem.

Reply:
The problem is the component can't assume that; `raw_devices_ordered` is just the "raw" value on the `SchoolDeviceAllocation` (i.e. not taking into account vcap pools) – if this component is ever reused in the future that assumption won't necessarily hold. Best to use `raw_devices_ordered`, I think. (Of course, the underlying "issue" is the naming and way vcap pools are done – but that's for another day :D)